addons/website_sale/static/src/js/product_template_attribute_line/product_template_attribute_line.xml

```diff
@@ -13,6 +13,16 @@
                 <div t-out="getReadOnlyPtal()" class="text-muted small"/>
             </t>
         </div>
+        <xpath expr="//input[@class='o_input w-lg-75 mb-4 ms-lg-auto']" position="replace">
+            <input
+                class="o_input w-lg-75 mb-4 ms-lg-auto"
+                type="text"
+                placeholder="Enter a customized value"
+                t-if="hasPTAVCustom &amp;&amp; isSelectedPTAVCustom()"
+                t-att-value="this.props.customValue"
+                t-on-change="updateCustomValue"
+            />
+        </xpath>
```

Comment:
Changing the DOM or even the style can easily break the xpath. It also creates a lot of code duplication, which is not good for maintenance: if someone updates any value of the original template and forgets to change this xpath, the content would not be the same depending on whether website_sale is installed

In my opinion, xpaths should always be a last resort, they're too hard to maintain and too easy to break

I'd suggest you move the placeholder value to a getter in the component and repeat it in website_sale instead

Reply:
Thanks, @oomsveta. This is better.